backend/routes/discord.py

```diff
@@ -63,14 +64,13 @@ async def delete(self, request: Request) -> JSONResponse:
 
         if member:
             return JSONResponse(member.dict())
-        else:
-            return NOT_FOUND_EXCEPTION
+        return NOT_FOUND_EXCEPTION
```

Comment:
I think the explicit else is more readable. At the least there should be a blank line after the if block.

Reply:
I'm going to flip this around so instead we do `if not member: ...` and then default to returning the found member, should help clarity.